### PR TITLE
Update RSAWEB copy

### PR DESCRIPTION
**Pull request by imsickofmaps**
Since being established in 2001, RSAWEB has grown rapidly, gaining extensive industry experience while continuously improving on, and introducing new products including our latest offerings; <a href="http://www.rsaweb.co.za/hosting/flex-cloud/">Flex Cloud</a> and <a href="http://www.rsaweb.co.za/connectivity/fibre/">City Fibre</a>. With great commitment to customer service excellence and in-depth technical understanding, RSAWEB is able to retain partnerships with some of South Africa’s leading brands, from a wide array of sectors. RSAWEB takes pride in offering custom made solutions to clients as well as the best in service and support, providing Service-Level Agreements and a 24-hour helpdesk that fully supports and monitors all services, ensuring all clients have absolute peace of mind.
Having invested heavily in infrastructure, RSAWEB has built a strong presence in South Africa and Internationally with data centres in Johannesburg and Cape Town, and POPs (points of presence) in London, Europe and the USA.
For more information visit us at www.rsaweb.co.za, or follow us on Twitter at www.twitter.com/RSAWEB, and Like Us on Facebook at www.facebook.com/RSAWEB

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@vhata review plz

File: _data/sponsors.yml [hunk 1]
```diff
@@ -203,31 +203,30 @@ profiles:
...
-      </p>
-      <p>
-      We believe that great technology doesn't have to be complex and expensive;
-      our motto 'technology meets life' implies just that. We bridge the gap
-      between IT innovation and everyday life - and our team take pride in doing
-      this every day.
-      </p>
+      Since being established in 2001, RSAWEB has grown rapidly, gaining
+      extensive industry experience while continuously improving on, and
+      introducing new products including our latest offerings;
```
Comment (vhata) [inline _data/sponsors.yml]: use ":" not ";"

File: _data/sponsors.yml [hunk 2]
```diff
@@ -203,31 +203,30 @@ profiles:
...
+      With great commitment to customer service excellence and in-depth
+      technical understanding, RSAWEB is able to retain partnerships with
+      some of South Africa’s leading brands, from a wide array of sectors.
+      RSAWEB takes pride in offering custom made solutions to clients as well
+      as the best in service and support, providing Service-Level Agreements
+      and a 24-hour helpdesk that fully supports and monitors all services,
+      ensuring all clients have absolute peace of mind.
+      </p><p>
+      Having invested heavily in infrastructure, RSAWEB has built a strong
+      presence in South Africa and Internationally with data centres in
```
Comment (vhata) [inline _data/sponsors.yml]: lower case 'i' on "internationally"

File: _data/sponsors.yml [hunk 3]
```diff
@@ -203,31 +203,30 @@ profiles:
...
+      Johannesburg and Cape Town, and POPs (points of presence) in London,
+      Europe and the USA.
+      </p><p>
       <p>
       For more information, visit <a href='http://www.rsaweb.co.za/'>
-      rsaweb.co.za</a>, and follow us on Twitter 
-      <a href='https://twitter.com/RSAWEB'>@RSAWEB</a>
+      rsaweb.co.za</a>, or follow us on Twitter
+      <a href='https://twitter.com/RSAWEB'>@RSAWEB</a>, and Like Us on Facebook
```
Comment (vhata) [inline _data/sponsors.yml]: lower case "like us"

**Comment by vhata** [comment]
:+1:

**Pull request merged**